projects/experimental/directives/progress-segmented/progress-segmented.directive.ts

```diff
@@ -0,0 +1,23 @@
+import {Directive, Inject, Input} from '@angular/core';
+import {TuiDirectiveStylesService} from '@taiga-ui/cdk';
+
+import {TuiProgressSegmentedComponent} from './progress-segmented.component';
+
+@Directive({
+    selector: '[tuiProgressBar][segments]',
+    host: {
+        class: '_segmented',
+        '[style.--t-segment-width]': '1 / segments',
+        '[attr.new]': '""', // TODO: drop in v4.0
```

Comment:
Do you use it anywhere?

Reply:
https://github.com/taiga-family/taiga-ui/blob/085538cf4a656f612a272ef1b1a8dd9f485cd303/projects/kit/components/progress/progress-bar/progress-bar.component.less#L65-L66

Reply:
https://taiga-ui.dev/components/progress-bar#sizes